step_03/instructions.md

```diff
@@ -1,10 +1,15 @@
-# Animate look using AnimatedContainer
+# Animate the box using AnimatedContainer
 
 Now is the time to finally create some animations!
 
 The widget already changes its state, so now the last step is to replace 
 `Container` with an `AnimatedContainer`. 
 
+<!-- Not to be picky, but would it make sense to make the duration configurable 
```

Comment:
@brianegan Thanks for the suggestion! I think it makes sense and this is how I would probably do in the production app :) However... I think that in that case it might be a little bit confusing and (as you said) distracting from the goal, so I'd rather keep it simple :) Thanks, though!